playwright.config.ts

```diff
@@ -0,0 +1,95 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  // testDir: "./tests",
+  // Glob patterns or regular expressions that match test files.
+  testMatch: "tests/**/*.spec.ts",
+  globalSetup: "./global-setup",
+  globalTeardown: "./global-teardown",
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  // workers: process.env.CI ? 1 : undefined,
+  workers: 3,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: "html",
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    httpCredentials: {
+      username: "guest",
+      password: "welcome2qauto",
+    },
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: "http://127.0.0.1:3000",
+    viewport: {
+      width: 1200,
+      height: 840,
+    },
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: "on-first-retry",
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: "setup",
+      testMatch: "**/setup/**/*.setup.ts",
+    },
```

Comment:
I do not see any setup files in your PR and you do not use it as dependency for now. So either comment it or remove